.github/workflows/docs.yml

```
@@ -0,0 +1,76 @@
+name: Docs
+
+on:
+  pull_request:
```

Let's keep it to manual triggers and merges to main when publishing to the public websites for now, until we can deploy to an ephemeral instance on PRs. 

```suggestion
  workflow_dispatch:
```